docs/docs-beta/docs/guides/build/assets/asset-jobs.md

```diff
@@ -24,29 +23,28 @@ You can target one or multiple assets, or create multiple jobs that target overl
 - `all_assets_job` targets all assets
 - `sugary_cereals_job` targets only the `sugary_cereals` asset
 
-{/* Original code example in https://github.com/dagster-io/dagster/blob/master/examples/docs_snippets/docs_snippets/concepts/assets/build_job.py between start_marker and end_marker */}
-<CodeExample filePath="" language="python" lineStart="" lineEnd=""/>
+<CodeExample filePath="guides/data-modeling/asset-jobs/asset-jobs.py" language="python" lineStart="20" lineEnd="33"/>
 
 ## Making asset jobs available to Dagster tools
 
 Including the jobs in a [`Definitions`](/api/python-api/definitions) object located at the top level of a Python module or file makes asset jobs available to the UI, GraphQL, and the command line. The Dagster tool loads that module as a code location. If you include schedules or sensors, the [code location](/guides/deploy/code-locations) will automatically include jobs that those schedules or sensors target.
 
 ```python file=/concepts/assets/jobs_to_definitions.py
-from dagster import Definitions, MaterializeResult, asset, define_asset_job
+import dagster as dg
```

Comment:
Updated the hardcoded example to match format. Not sure if this should remain unchanged or be moved to a `CodeExample`

Reply:
We can keep it here for now, and will refactor in a follow-up, ty.